pkg/utils/nodeclass/suite_test.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("NodeClassUtils", func() {
 	})
 	It("should convert a AWSNodeTemplate to a EC2NodeClass (with AMISelector name and owner values set)", func() {
 		nodeTemplate.Spec.AMISelector = map[string]string{
-			"aws::name":   "ami-name1,ami-name2",
+			"aws::name":   "ami-name1,ami-name2, ami-name3",
```

Comment:
I would expect this should require a change further down in the test when you check the amiselector terms, but I don't see a change.